view/library/send-to-editor.php

```diff
@@ -207,7 +213,9 @@ function kaltura_updateRatio() {
 				entryId    : <?php echo wp_json_encode($this->entry->id); ?>,
 				playersList: '#uiConfId',
 				dimensions : 'input[name=playerRatio]',
-				submit     : 'input[name=sendToEditorButton]'
+				submit     : 'input[name=sendToEditorButton]',
+				entryError: '<?php echo $this->entryError; ?>',
```

Comment:
Fixed.
While testing, I discovered another bug and fixed it. Please see https://github.com/kaltura/all-in-one-video-pack.wordpress/pull/47/commits/73b079aea1a46859ddf013a22c3ae4c99d14d2fd as well. :)